.github/workflows/check-submodule-pointers.yml

```
@@ -0,0 +1,32 @@
+name: Check submodule pointers
+
+on:
+  push:
```

Also, nice to have a `workflow_dispatch` trigger so we can manually run this for testing or anything else. 
```suggestion
  workflow_dispatch:
  push:
```